### PR TITLE
docs/unified-test-documentation/dasharo-security/201-verified-boot.md: add VBO011.001 test case

### PR DESCRIPTION
**Pull request by Pokisiekk**

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-security/201-verified-boot.md [hunk 1]
```diff
@@ -487,3 +487,57 @@ command:
...
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Download `cbmem` and `flashrom` from <https://cloud.3mdeb.com/index.php/s/zTqkJQdNtJDo5Nd>
+   to the DUT.
+1. Disable Secure Boot.
+1. Slot A is flashed with an binary with wrong vboot keys as in test case
+   `VBO008.001`.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/201-verified-boot.md:512]: This point suggest, that we need to do the test. Couldn't we limit ourselves to saying that the device is currently being flashed with a binary with the wrong keys?

File: docs/unified-test-documentation/dasharo-security/201-verified-boot.md [hunk 2]
```diff
@@ -487,3 +487,57 @@ command:
...
+
+**Expected result**
+
+1. Popup with information about recovery mode should not be displayed.
+1. The logs should indicate that vboot hasn't chosen to boot from the recovery
+   slot. Example output:
+
+    ```bash
+    VB2:vb2_check_recovery() Recovery reason from previous boot: 0x0 / 0x0
+    ```
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/201-verified-boot.md:543]: @Pokisiekk and how about actualize the test-matrix? This test is dedicated to all of platforms?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-security/201-verified-boot.md (hunk 1 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-security/201-verified-boot.md]: Sure, done

File: docs/unified-test-documentation/dasharo-security/201-verified-boot.md [hunk 3]
```diff
@@ -487,3 +487,56 @@ command:
...
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Download `cbmem` and `flashrom` from <https://cloud.3mdeb.com/index.php/s/zTqkJQdNtJDo5Nd>
+   to the DUT.
+1. Disable Secure Boot.
+1. Slot A is flashed with an binary with wrong vboot keys.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-security/201-verified-boot.md:511]:
```suggestion
1. Slot A is flashed with an binary with wrong-signed vboot keys.
```

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**